wormhole/proxy/wormhole_test.go

```diff
@@ -62,133 +57,3 @@ func TestClientRetry(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "success", creds.Host)
 }
-
-func TestProxiedClient(t *testing.T) {
```

Comment:
These tests aren't running in CI, which IMO makes them kind of useless - if someone needs them again, they can pull them out of the git history.